internal/deps/composition_test.go

```diff
@@ -18,6 +18,23 @@ func TestDeploymentOrderNoDeps(t *testing.T) {
 	require.ElementsMatch(t, []string{"app1", "app2"}, order)
 }
 
+func TestCompositionContains(t *testing.T) {
+	comp := NewComposition()
+	comp.Add("prd", "app1", &Dependencies{})
+
+	exists, err := comp.Contains("abc", "app1")
```

Comment:
should we have positive test case for one item only?

Reply:
I added another distribution + another app to the composition.